action.yml

```diff
@@ -107,6 +107,7 @@ runs:
       with:
         cache: 'pnpm'
         node-version: ${{ inputs.node-version }}
+        registry-url: ${{ inputs.node-registry-url }}
```

Comment:
At the veeery top of this file, can you add a description for this new input? Like node-version has?

Thanks!

Reply:
Oops ! I added the input entry :)